tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -33,3 +33,22 @@ def test_zos_job_query_func(ansible_zos_module):
     for result in results.contacted.values():
         assert result.get("changed") is False
         assert result.get("jobs") is not None
+
+
+# test to show multi wildcard in Job_id query won't crash the search
+def test_zos_job_query_multi_wildcards_func(ansible_zos_module):
+    hosts = ansible_zos_module
+    results = hosts.all.zos_job_query(job_id="STC*3*")
+    pprint(vars(results))
+    for result in results.contacted.values():
+        assert result.get("changed") is False
+        assert result.get("jobs") is not None
+
+# test to show multi wildcard in Job_name query won't crash the search
+def test_zos_job_query_multi_wildcards_func(ansible_zos_module):
```

Comment:
I don't see  a job submitted thus we are relying on the ones on the system which works for now but not for portability in the future, but this is easy enough to fix. What caught my eye was the  `job_name="JOB*1*"` seems to be a JOB ID, i logged on to my system to find a job name starting with JOB but all I found was Job IDs. Could this be an error?

Reply:
Yes..Fernando mentioned this as well... the new tests run a copy of the hello jcl.  Job ID takes out 2 characters and uses asterisks for search, while job name search looks for "HE*L*".... That should go through the pipeline this evening.